src/blueprints/tv2/value-objects/tv2-show-style-blueprint-configuration.ts

```diff
@@ -1 +1,19 @@
-export interface Tv2ShowStyleBlueprintConfiguration {}
+export interface Tv2ShowStyleBlueprintConfiguration {
+  GfxDefaults: GfxDefault
```

Comment:
Yes, unforutnately these need to match the name in the database which is currently is `GfxDefaults`.
The reason for this is because of the "framework" Core is exposing to Blueprints to do UI where we had to use a "Table".
This means that the values of `GfxDefaults` is saved as an array to the database by Core.
But since we enforce that there can only ever be one `GfxDefaults` we map the array to the a single unit when converting the "unknown" configuration parsed from SofieServer to Blueprints into a Tv2Configuration.

When we control settings, we have the option to change it.